editor/store/effects.js

```diff
@@ -302,7 +302,7 @@ export default {
 		const { postId } = action;
 		const basePath = wp.api.getPostTypeRoute( getCurrentPostType( getState() ) );
 		dispatch( removeNotice( TRASH_POST_NOTICE_ID ) );
-		apiRequest( { path: `/wp/v2/${ basePath }/${ postId }`, method: 'DELETE' } ).then(
+		fetch( { path: `/wp/v2/${ basePath }/${ postId }`, method: 'DELETE' } ).then(
```

Comment:
In the error handler for this request (line 317) we reference `err.responseJSON` which will now be undefined. We do this in most places that used `apiRequest`.